coalib/bears/requirements/PackageRequirement.py

```diff
@@ -49,3 +49,38 @@ def check(self):
         :return: Returns True if satisfied, False if not.
         """
         raise NotImplementedError
+
+    @classmethod
+    def multiple(cls, *args):
```

Comment:
I'm personally a fan of the 3rd option as it has pretty good run time checks included that you would have to implement yourself otherwise. Embrace Python 3 👍